### ISSUE TITLE
Request: IDA Pro IDB file

### ISSUE THREAD

**Issue by Piruzzolo** [FolkertVanVerseveld/aoe#21]
Hi, I'm a big fan of AoE and really like your project. I'm contributing to [this](https://github.com/yvan-burrie/Genie-Reverse) project, which is a reverse-engeneering effort of the Genie game engine. Since that's part of your work on this big project, I thought it was better for us to share as much as possible, for you to check your port's behaviour against the actual decompiled code and for us, to integrate your findings and decompiled/understood functions inside the code. For example, I really liked [this one](https://github.com/FolkertVanVerseveld/aoe/blob/master/tools/cheat.c). 

So, if you want, you could share with us the IDB file you're working on.
And keep up the good work, man :smile:

**Comment by FolkertVanVerseveld** [comment]
Hello Marco,

Glad to see you like my project! I think I've seen that project before
you are contributing too. To be honest I wasn't sure back then if I
could help you out so I sticked to my own project, but now I can see we
can all benefit if we start working together.

It has been a long time I was reverse-engineering and I didn't get as
far as you guys got, so I'm still not sure if I can provide much help.

I think we can all really benefit if we can build a stable and good
foundation on which all AoE related projects can build (e.g. openage,
which I've contributed to some time ago).

Are you interested in play-testing or building core mechanics as well?
As far as I can see, you are mainly focused on reverse-engineering.

My focus initially started on exactly replicating the engine's behavior,
but then I started to think this may get me in legal trouble if I would
replicate *all* behavior, which is clearly not needed for user interface
stuff amongst other things.

As time went on, I've only replicated the necessary things to be able to
create the game (i.e. DRS and related file format parsing), but I've
never touched the game logic since I got stuck trying to
reverse-engineer it.

I'll look at my idbs and see if I can clean them up, but I don't it is
really usuful input :-P. Let me know what you think and lets keep in
touch!
